### DISCUSSION TITLE
[HTTP]:500 - [CorrelationId]:aa3076a0-d0af-2000-cb87-77d2da1f7054 [Version]:16.0.0.23019

### DISCUSSION THREAD

**Discussion by GangadharMFord** [microsoft-search/pnp-modern-search#2516] [Q&A] [answered]
Hi Team,

I have facing HTTP: 500 correlation error for few searches keywords.
Cross checked the network traces and found the below postquery is failing with error code 500 and please find the below details as below.
Request Method: POST
Status Code: 500
Response header:
Content security-policy: frame-ancestors 'self' teams.microsoft.com *.teams.microsoft.com *.skype.com *.teams.microsoft.us local.teams.office.com *.powerapps.com *.yammer.com *.officeapps.live.com *.office.com *.stream.azure-test.net *.microsoftstream.com *.dynamics.com *.microsoft.com securebroker.sharepointonline.com;
nel: {"report_to":"network-errors","max_age":7200,"success_fraction":0.001,"failure_fraction":1.0}

{"group":"network-errors","max_age":7200,"endpoints":[{"url":"https://spo.nel.measure.office.net/api/report?tenantId=XXXXXXXXXXXXXXXX&destinationEndpoint=Edge-Prod-CH1r5e&frontEnd=AFD"}]}

{
"code": "-1, Microsoft.Office.Server.Search.REST.SearchServiceException",
"message": {
"lang": "en-US",
"value": "An unknown error occurred."
}
}

Network response :
{"odata.error":{"code":"-1, Microsoft.Office.Server.Search.REST.SearchServiceException","message":{"lang":"en-US","value":"An unknown error occurred."}}}

please let us know if you need a browser networking logs for analysis 

NOTE: When I start disbaling query rules option then it starts working with no error for the same keyword.

**Comment by wobba** [comment, marked as answer]
@GangadharMFord do you need to use query rules? If not leave them off. The issue likely appears when triggering the default query rule for people results used in classic search experiences. Or you can make it inactive to see if it solves your issue.

![image](https://user-images.githubusercontent.com/51104/200293332-5921e94c-b378-4a6d-b848-8b289c3d0e6a.png)

**Reply by GangadharMFord** [reply]
Thank you @wobba, Its worked. thank you again for your help with quick response.
